website/content/docs/concepts/stateful-deployments.mdx

```diff
@@ -38,19 +37,19 @@ job "app" {
 }
 ```
 
-If a volume is set to be sticky, during the deployment the scheduler associates
-its ID with the task group that uses it. Stateful deployments require that any
-allocations belonging to that task group will only be (re)placed on nodes that
-have this volume ID available.
+If you set the volume to "sticky," during the deployment the scheduler
+associates its ID with the task group that uses it. Stateful deployments require
+that any allocations belonging to that task group will only be (re)placed on
+nodes that have this volume ID available.
 
 Scaling jobs with sticky volumes up results in more volume IDs claimed by the
 task group, but scaling jobs down does not delete unused volumes, nor does it
 touch the data present on them.
 
 If the scheduler cannot find a node that has the right volume ID present
-(perhaps because the node is down or disconnected), a blocked evaluation will be
-created and the job will remain in `pending` status until it's either stopped by
-the user or a feasible node is found.
+(perhaps because the node is down or disconnected), a blocked evaluation is
+created and the job remains in `pending` status until it's either stopped by the
```

Comment:
I'd consider leaving out the note about the job being in pending status. There are users who read docs at an extremely literal level and will not be able to reason that this clause only applies in the case of an actively deploying job and not, ex. a rescheduled alloc.

Reply:
[19b2c7b](https://github.com/hashicorp/nomad/pull/25288/commits/19b2c7bbe281582ce1a1956afa6bdfedef8c0a53)